books-call-number/nodes/email.json

```
@@ -0,0 +1,33 @@
+{
+  "id": "2fff188a-9bf3-43f7-939f-fcce0c4387dd",
+  "name": "Email checked-out items list",
```

I think we can still shorten this further.
```suggestion
  "name": "E-mail Items",
```

---

Correction in place